python/revng/internal/pipeline.yml

```diff
@@ -91,6 +91,9 @@ branches:
         arguments: [llvm-root, cfg-map]
       - savepoint: cfg-computed
         containers: [cfg-map]
+        artifacts:
+          - name: emit-cfg
+            container: cfg-map
```

Comment:
I'm not entirely sure what's the value of having all these micro commits on `python/revng/internal/pipeline.yml`.

Can we have a single commit at the end?